src/connectedk8s/azext_connectedk8s/commands.py

```diff
@@ -19,6 +19,7 @@ def load_command_table(self, _):
 
     with self.command_group('connectedk8s', connectedk8s_sdk, client_factory=cf_connected_cluster) as g:
         g.custom_command('connect', 'create_connectedk8s', supports_no_wait=True)
+        g.custom_command('update', 'update_agents', supports_no_wait=True)
```

Comment:
I dont see the use of no_wait in the update_agents function. We can remove this parameter if thats the case.